api/TypeDBOptions.java

```diff
@@ -42,6 +44,8 @@
     private Integer sessionIdleTimeoutMillis = null;
     private Integer schemaLockAcquireTimeoutMillis = null;
 
+    private TypeDBOptions() { }
```

Comment:
Prevent users from accidentally calling `new TypeDBOptions`. The class must be instantiated with either `TypeDBOptions.core()` or `TypeDBOptions.cluster()`.